src/hooks/useDisableScroll/__stories__/useDisableScroll.stories.mdx

```diff
@@ -0,0 +1,60 @@
+import { Canvas, Meta, Story } from "@storybook/addon-docs";
+import useScrollLock from "..";
+import { FunctionArgument, FunctionArguments, UsageGuidelines } from "../../../storybook";
+
+<Meta title="Hooks/useDisableScroll" />
+
+# useScrollLock
+
+- [Overview](#overview)
+- [Returns](#returns)
+- [Usage](#usage)
+- [Feedback](#feedback)
+
+## Overview
+Use this hook to disable scroll for element.
+
+<Canvas>
+  <Story name="Overview">
+    {() => {
+        const { disableScroll, enableScroll } = useScrollLock(".scrollable");
```

Comment:
If the name of the hook is use disable scroll let's call it the same way in the example? It will be more clearer to readers this way

Reply:
Opps, I rename the hook and miss this one